README.rst

```diff
@@ -20,6 +20,13 @@ To install:
 Release History
 ---------------
 
+2021-01-26 Version 0.6.21
++++++++++++++++++++++++++
+
+**Bug Fixes**
+
+- Fixes `failsafe_deserialize`. **Please bump your version to 0.6.21** #232
```

Comment:
Until someone uses the new autorest, they won't call this methd, so they won't fail. Therefore, I would keep it simple and less scary:

> Fixes "failsafe_deserialize" introduced in 0.6.20